test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.cs

```diff
@@ -300,7 +300,39 @@ partial class C
                 static partial void M1(ILogger logger, int param);
             }}";
 
-        await RunGenerator(source, DiagDescriptors.TemplateStartsWithAtSymbol);
+        await RunGenerator(source);
+    }
+
+    [Theory]
```

Comment:
In general, test cases that validate the happy path should be listed in the TestClasses, and then the generated code for those can be validated in the Generated/* test cases.

Otherwise, this test right now is validating that the generator doesn't fail, but it's not checking that the generated code is actually correct.

Reply:
There's added tests for this under AtSymbolsTestExtensions.cs

Reply:
Then you can delete these :-)

The idea is that anything that compiles is validated via the GeneratedTests, and the only things that get tested by the parser tests are things that don't compile, where we check the error handling behavior.

Reply:
Hmm, with parser tests also providing the GeneratedSourceResult we can additionally test the actual source being generated and use Verify against it.

Reply:
Thinking about it, the emitter tests could do this instead. I have some things to handle and will return back to the PR in a few days.